### ISSUE TITLE
Any straightforward way to use the lib for two data sets on the same graph

### ISSUE THREAD

**Issue by nikitph** [rainbow-me/react-native-animated-charts#49]
![image](https://user-images.githubusercontent.com/5007879/142981008-b40397f2-d796-4d81-aa8a-b9ed9c245e08.png)

we are trying to achieve the above

**Comment by ajimae** [comment]
I don't think this chart has the functionality, checkout [this chart library ](https://github.com/coinjar/react-native-wagmi-charts#line-chart)it has the this very functionality I have tested it and it's working fine.
